lambda-kernel/shared/src/main/scala/feral/lambda/Context.scala

```diff
@@ -87,7 +119,8 @@ object Context extends ContextCompanionPlatform {
       logStreamName: String,
       identity: Option[CognitoIdentity],
       clientContext: Option[ClientContext],
-      remainingTime: F[FiniteDuration]
+      remainingTime: F[FiniteDuration],
+      xRayTraceId: Option[String]
```

Comment:
propagate `X-Ray-Trace-Id` to handlers via `Context` to workaround the limitation that environment variables couldn't be set from JVM runtime.